phpBB/phpbb/notification/method/messenger_base.php

```diff
@@ -112,22 +112,29 @@ protected function notify_using_messenger($notify_method, $template_dir_prefix =
 				continue;
 			}
 
-			$messenger->template($notification->get_email_template(), $user['user_lang'], '', $template_dir_prefix);
-
-			$messenger->set_addresses($user);
-
-			$messenger->assign_vars(array_merge(array(
-				'USERNAME'						=> $user['username'],
-
-				'U_NOTIFICATION_SETTINGS'		=> generate_board_url() . '/ucp.' . $this->php_ext . '?i=ucp_notifications&mode=notification_options',
-			), $notification->get_email_template_variables()));
-
-			$messenger->send($notify_method);
+			/** @psalm-suppress InvalidTemplateParam */
```

Comment:
What are you suppressing here and why?

Reply:
Because psalm throws error without this, and I don't already remember what error because the commits tests history is lost already.